data/spells/scripts/support/animate_dead_rune.lua

```diff
@@ -18,6 +18,6 @@ function onCastSpell(creature, variant, isHotkey)
 	end
 
 	creature:getPosition():sendMagicEffect(CONST_ME_POFF)
-	creature:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
+	creature:sendTextMessage(MESSAGE_STATUS_SMALL, Game.getReturnMessage(RETURNVALUE_NOTPOSSIBLE))
```

Comment:
[Player.sendCancelMessage](https://github.com/otland/forgottenserver/blob/master/data/lib/core/player.lua#L58-L63) is not a compatibility function though, and it's widely used in the data pack & source code.